fiftyone/server/lightning.py

```
@@ -181,7 +181,7 @@ class DistinctQuery:
 def _resolve_lightning_path_queries(
     path: LightningPathInput, dataset: fo.Dataset, info: Info
 ) -> t.Tuple[
-    AsyncIOMotorCollection,
+    AsyncCollection,
     t.Union[DistinctQuery, t.List[t.Dict]],
```

Ruff (0.16.1)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)